spec/webmock/user_spec.rb

```diff
@@ -21,5 +21,34 @@
       # Assert the response and test your code's behavior
       expect(user['email']).to eq('mb@example.com')
     end
+
+    ###################################################################################################################
+    # GET USER TEST
+    # #################################################################################################################
+    it 'Fetches a list of Users' do
+      body = [
+        {
+          "email" => "mb@example.com",
+          "first_name"=> "Admin",
+          "id"=> 1,
+          "last_name"=> "User",
+        },
+        {
+          "email" => "mb1@example.com",
+          "first_name"=> "Admin1",
+          "id"=> 2,
+          "last_name"=> "User1",
+        }
+      ]
+
+      stub_request(:get, 'http://localhost:3030/api/user')
+        .to_return(status: 200, body: body.to_json)
+
+      user_json = client.users
+      users = JSON.parse(user_json)
+
+      # Assert the response and test your code's behavior
+      expect(users[0]['email']).to eq('mb@example.com')
```

Comment:
consider adding the following test to check that the request is not returning duplicate users
```ruby
expect(users.length).to eq(2)
```

Reply:
considered, thanks for the recommendation!